lib/plugins/aws/logs/README.md

```diff
@@ -0,0 +1,42 @@
+# Logs
+
+This plugin returns the CloudWatch logs of a lambda function. You can simply run `serverless logs -f hello` to test it out.
+
+## How it works
+
+`Logs` hooks into the [`logs:logs`](/lib/plugins/logs) lifecycle. It will fetch the CloudWatch log group of the provided function and outputs all the log stream events in the terminal.
+
+## Logs Options
+
+The logs plugin only require the function name you want to fetch the logs for. Other than that there are other options that you can add for extra control:
+
+* **stage:** (shortcut: `-s`) - The stage you want to view the function logs for. If not provided, the plugin will use the default stage listed in `serverless.yaml`. If that doesn't exist either it'll just fetch the logs from the `dev` stage.
```

Comment:
We have used something like "- `--template` or `-t` The name of your new service. **Required**." in the other plugin docs. Should be consistent here.